command.go

```diff
@@ -783,3 +783,92 @@ func (cmd *ClusterSlotCmd) parseReply(cn *conn) error {
 	cmd.val = v.([]ClusterSlotInfo)
 	return nil
 }
+
+//------------------------------------------------------------------------------
+
+// Location type for GEO operations in Redis
+type GeoLocation struct {
+	Longitude, Latitude, Name, Distance string
+	GeoHash int64
+}
+
+type GeoCmd struct {
+	baseCmd
+
+	locations []GeoLocation
+}
+
+// Query type for geo radius
+type GeoRadiusQuery struct {
+	Key string
+	Longitude, Latitude, Radius string
+	// Unit default to km when nil
+	Unit string
+	WithCoord, WithDist, WithHash bool
```

Comment:
WithCoordinates, WithDistance, WithGeoHash (will let the 'with' prefix stay for bool)